gradle/component.gradle

```diff
@@ -15,6 +15,7 @@ kotlin {
                 jvmTarget = "1.8"
                 freeCompilerArgs += [
                         '-Xopt-in=kotlin.ExperimentalUnsignedTypes',
+                        '-Xopt-in=kotlin.ExperimentalStdlibApi',
```

Comment:
I think this can just be done [here] like the others (https://github.com/splendo/kaluga/blob/6a8bba0efb8d0fb17e60eec6342418ec877c9b50/gradle/component.gradle#L143)

And then of course the annotations are no longer needed in the code

Reply:
Good point, moved all flags in one place